xml/Sector.xml

```diff
@@ -1072,7 +1082,7 @@
 				<narrative>Narcotics control</narrative>
 			</name>
 			<description>
-				<narrative>In-country and customs controls including training of the police; educational programmes and awareness campaigns to restrict narcotics traffic and in-country distribution.</narrative>
+				<narrative>In-country and customs controls including training of the police; educational programmes and awareness campaigns to restrict narcotics traffic and in-country distribution 1.</narrative>
```

Comment:
this is a superscript ‘1’ i.e. there’s a footnote somewhere that it refers to. It should probably be removed.